tests/spec/s-xregexp.js

```diff
@@ -506,49 +507,6 @@ describe('XRegExp()', function() {
 
         });
 
-        describe('explicit numbered backreferences', function() {
```

Comment:
All of the removed tests in this block are still correct and relevant. Please restore them.